etna/collections/models.py

```diff
@@ -244,14 +244,22 @@ def get_datalayer_data(self, request: HttpRequest) -> Dict[str, Any]:
 
     @cached_property
     def related_articles(self):
-        from etna.articles.models import ArticlePage
+        """Return a list of related pages for rendering in the related articles section
+        of the page. To add another page type, add it to the `exact_type` query below.
+        """
+
+        from wagtail.models import Page
+
+        from etna.articles.models import ArticlePage, FocusedArticlePage
 
         return (
-            ArticlePage.objects.exclude(pk=self.featured_article)
+            Page.objects.exclude(pk=self.featured_article.pk)
```

Comment:
I believe this will raise an AttributeError if `self.featured_article` is `None` (which it can be)

Reply:
What you could do though, is `self.featured_article_id`

Reply:
Hi Andy, I tried removing a featured article on a page and it seems to have worked :)

Reply:
Honestly, I would use `self.featured_article_id` here. It's something Django provides for free for every FK field, and the value is always going to be `None` or and `int` - which is what we want.